frontend/src/components/core/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -161,8 +161,12 @@ function aboutDialog(props: AboutProps): ReactElement {
       <ModalHeader>Powered by</ModalHeader>
       <ModalBody>
         <div>
-          Streamlit v{SessionInfo.current.streamlitVersion}
-          <br />
+          {SessionInfo.isSet() && (
```

Comment:
Can we add a comment above here explaining why this is necessary? Something like

```
// Show our version string only if Streamlit has connected to the backend and gotten a valid SessionInfo
```